tests/mig/mig_test.go

```
@@ -52,6 +49,12 @@ var _ = Describe("MIG", Ordered, Label(tsparams.LabelSuite), func() {
 			// Initialize CLI flag-derived values after flags are parsed
 			mig.ParseCLIParameters()
 			mig.LogCLIParameterValues()
+
+			WorkerNodeSelector = map[string]string{
+				inittools.GeneralConfig.WorkerLabel:                          "",
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)